template/web/app/paymaster-bundler/layout.tsx

```diff
@@ -1,8 +1,9 @@
 import { generateMetadata } from '@/utils/generateMetadata';
 
 export const metadata = generateMetadata({
-  title: 'Build Onchain Apps Toolkit',
-  description: 'Build Onchain Applications with the best consumer experience in a few minutes.',
+  title: 'Paymaster - BOAT',
```

Comment:
Let's do `Paymaster Bundler`

Reply:
@Zizzamia should we be spelling out `BOAT` or keep it as an acronym?